tests/acceptance/lazy-routeable-engine-test.js

```diff
@@ -74,6 +105,40 @@ test('it should pause to load JS and CSS assets on an initial transition into a
   andThen(() => verifyInitialBlogRoute(assert, this.loadEvents, this.application));
 });
 
+test('it should pause to load JS and CSS assets on deep link into a lazy Engine after retrying the transition', function(assert) {
```

Comment:
Its not immediately obvious to me how this test is supposed to differ from the other test being added just above, can you modify the test descriptions to make it clearer?

Reply:
Something like `it should pause to load JS and CSS assets if previous deep link into a lazy Engine has failed` could be better?

Reply:
Yes, that sounds better to me